actions/setup-sigstore-env/build-trusted-root.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+#
+# Copyright 2025 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Build a Sigstore trustedroot.json and signingconfig.json using a running test instance and source dirs
+#
+# Requires cosign binary to be in PATH.
+# Options can be given multiple times (signing config will contain only the last URLs,
+# but trustedroot will contain every instance):
+#  --fulcio <INSTANCEURL> <KEYFILE>
+#  --rekor-v1-url <INSTANCEURL>
+#  --rekor-v2 <INSTANCEURL> <KEYFILE>
+#  --timestamp-url <INSTANCEURL>
+
+set -euo pipefail
+
+
+[ -f trusted_root.json ] || [ -f signing_config.json ] && echo "trusted_root.json or signing_config.json already exist" && exit 1
+
+CMD="cosign trusted-root create"
```

Comment:
Could we use the containerized version of Cosign so that we know Cosign is available and avoid prompting a user to install Cosign?

Reply:
got it working.